test/Libraries/RevitIntegrationTests/SelectionTests.cs

```diff
@@ -456,6 +456,11 @@ public void DynamoAllSelectionNodeTests_WithPreSelectedEntities()
                 Assert.IsNotNull(refPt);
             }
 
+            // ElementsOfCategories & Views, as output of Categories 
+            // passed to ElementsOFCategory
+            var elementsOfViewNodeID = "2569434a-b34f-4512-a4fe-f065c7a10175";
+            AssertPreviewCount(elementsOfViewNodeID, 33);
```

Comment:
Add a new case filtering out all views in the sample.rvt, since template views will appear null, by pass the null check.